vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -160,6 +167,21 @@ def parse_pod(pod)
         end
       end
 
+      # NOTE: what we are trying to access here is the attribute:
+      #   pod.metadata.annotations.kubernetes.io/created-by
+      # but 'annotations' may be nil. The weird attribute name is
+      # generated by the JSON unmarshalling.
+      createdby_txt = pod.metadata.annotations.try("kubernetes.io/created-by")
```

Comment:
@jrafanie I've added the tests but now I am depending on #3011, can you review/merge that first and then I'll re-push this one with the relevant tests. Thanks!